packages/front-end/pages/fact-metrics/[fmid].tsx

```diff
@@ -194,9 +194,12 @@ export default function FactMetricPage() {
     );
   }
 
+  // canUpdateMetric takes in the current metrics + any updates - canEdit is used to determine if we should show the "Edit" buttons, so pass in the existing metric twice
```

Comment:
Same as before, might make sense to pass an empty object as the 2nd argument instead